apps/poker-roll/src/app/graphComponent.jsx

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+
+import { ResponsiveLine } from '@nivo/line'
+
+const data = {
+    "id": "japan",
+    "color": "hsl(68, 70%, 50%)",
+    "data": [
+      {
+        "x": "plane",
+        "y": 131
+      },
+      {
+        "x": "helicopter",
+        "y": 277
+      },
+      {
+        "x": "boat",
+        "y": 204
+      },
+      {
+        "x": "train",
+        "y": 180
+      },
+      {
+        "x": "subway",
+        "y": 103
+      },
+      {
+        "x": "bus",
+        "y": 87
+      },
+      {
+        "x": "car",
+        "y": 61
+      },
+      {
+        "x": "moto",
+        "y": 54
+      },
+      {
+        "x": "bicycle",
+        "y": 201
+      },
+      {
+        "x": "horse",
+        "y": 21
+      },
+      {
+        "x": "skateboard",
+        "y": 200
+      },
+      {
+        "x": "others",
+        "y": 186
+      }
+    ]
+  };
+
+const Graph = ({ data }) => {
+    return (
+        <ResponsiveLine
+            data={data}
```

Comment:
@AMess33 Here's the error I'm seeing in the console:

```
Warning: Failed prop type: The prop `data` is marked as required in `We`, but its value is `undefined`.
```

Not sure why it's referring to the component as `We` but should give you a hint that something's going on with your `data` variable.

This is the type of stuff TypeScript is really good for. Notice you've got `data` defined in your Graph component props and also up above for your mock data. When passing `data` into `ResponsiveLine` here, it's using `data` from props (not actually being passed to `Graph`) because that's in scope. Does your editor show an underline on your mock data? That lets you know your variable isn't actually being used.

<img width="715" alt="Screenshot 2023-09-01 at 6 57 49 AM" src="https://github.com/AMess33/PokerRoll/assets/20647694/9c4defd6-c99f-4b91-a0ad-bfb16b8a68a3">

Reply:
After that, double check the shape of your mock data. Looks like you've got a single piece of data with multiple "points" but need a collection of data that the line chart can loop over.

Reply:
fixed the data variable issue. need to look into the graph more, getting a mapping error like you mentioned, need to see where thats taking place and how to fix it

Reply:
@AMess33 Notice the original example data was a more complex object but now you're just using `sampleData.data`. Double check the docs for the expected structure for the `data` prop.

<img width="356" alt="Screenshot 2023-09-02 at 8 04 03 AM" src="https://github.com/AMess33/PokerRoll/assets/20647694/f62cd696-92fc-443a-909f-e0ddd436bb26">

Reply:
needed to read through the graph docs to get it all straight. the graph is showing up as intended now using the sample data.